transactions/FlowServiceAccount/set_execution_effort_weights.cdc

```diff
@@ -38,8 +38,8 @@
 // ]
 // ```
 transaction(newWeights: {UInt64: UInt64}) {
-    prepare(signer: AuthAccount) {
-        signer.load<{UInt64: UInt64}>(from: /storage/executionEffortWeights)
-        signer.save(newWeights, to: /storage/executionEffortWeights)
+    prepare(signer: auth(Storage) &Account) {
+        signer.storage.load<{UInt64: UInt64}>(from: /storage/executionEffortWeights)
```

Comment:
Applies to all transactions where raw values are saved in storage - do we need any sort of input validation for these? I'd imagine the values are sensitive and probably live in storage due to contract update limitations, but adding validation in the committing transaction would at least *feel* a bit safer.

Getting out of scope a bit, but if we know what these values will remain and don't think more will be added in the future, we could move them to some kind of config resource or contract and add setters so the input validation is done at the contract level and we could emit events when the values are updated.